Cisco/cisco-secure-firewall/tests/test_ASA_106023.json

```diff
@@ -28,13 +29,18 @@
       "vendor": "Cisco"
     },
     "related": {
+      "hosts": [
+        "ACMEsL",
+        "identity"
+      ],
       "ip": [
         "10.0.200.29",
         "224.0.1.129"
       ]
     },
     "source": {
-      "address": "10.0.200.29",
+      "address": "ACMEsL",
+      "domain": "ACMEsL",
```

Comment:
I think it's the same here. It seems that this value is more suitable in [`observer.ingress.interface.name`](https://www.elastic.co/guide/en/ecs/8.16/ecs-interface.html#field-interface-name)